assets/js/environments.js

```diff
@@ -208,7 +210,7 @@ jQuery(function () {
       createTable({
         id: 'componentsEnvironmentTable',
         data: transformedData,
-        orderColumn: 0,
+        orderColumn: 1,
```

Comment:
More useful to sort by service name rather than env name.. don't want to see 200 "dev" envs in no other order